README.md

```diff
@@ -32,7 +32,7 @@ To install you can use the following commands
 
 
 ```
-conda create -n comododev -c conda-forge adam-robotics idyntree bipedal-locomotion-framework=0.15.0 mujoco mujoco-python numpy mujoco-python-viewer matplotlib urllib3 urchin
+conda create -n comododev -c conda-forge adam-robotics idyntree bipedal-locomotion-framework=0.16.0 mujoco mujoco-python numpy mujoco-python-viewer matplotlib urllib3 urchin resolve-robotics-uri-py
```

Comment:
Do we need to hardcode blf version? Can we just specify a minimum version?

Reply:
I prefer to be dependent on a specific version since blf is continuously changing and I do not have any CI yet to check that the code works properly